src/Arcus.WebApi.Tests.Unit/Hosting/TestApiServer.cs

```diff
@@ -112,18 +120,48 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
                 });
             });
 
-            builder.UseStartup<TestStartup>();
+            builder.Configure(app =>
+            {
+                foreach (Action<IApplicationBuilder> configure in _configures)
+                {
+                    configure(app);
+                }
+
+                app.UseMiddleware<ExceptionHandlingMiddleware>();
+                app.UseMiddleware<TraceIdentifierMiddleware>();
+
+                app.UseHttpCorrelation();
+                app.UseSerilogRequestLogging();
```

Comment:
We don't need that as that's what we are adding

Reply:
Oh, yes.